Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/UpdateEmoteInputSystem.cs

```diff
@@ -92,5 +103,34 @@ private void TriggerEmote([Data] int emoteIndex, in Entity entity, in Profile pr
 
             messageBus.Send(emoteId, false, true);
         }
+
+        private void ListenToSlotsInput(InputActionMap inputActionMap)
+        {
+            for (var i = 0; i < 10; i++)
+            {
+                string actionName = GetActionName(i);
+
+                try
+                {
+                    InputAction inputAction = inputActionMap.FindAction(actionName);
+                    inputAction.started += OnSlotPerformed;
+                    actionNameById[actionName] = i;
+                }
+                catch (Exception e) { ReportHub.LogException(e, new ReportData(GetReportCategory())); }
+            }
+        }
+
+        private void UnregisterSlotsInput(InputActionMap inputActionMap)
+        {
+            for (var i = 0; i <= 9; i++)
```

Comment:
in the for loop above we do `< 10` and here we do `<= 9` I think before this we had `< 10` and `< 8` haha I think its best to decide on one and keep it, something like `< Avatar.MAX_EQUIPPED_EMOTES`